tests/controls/infobar.cpp

```diff
@@ -0,0 +1,45 @@
+///////////////////////////////////////////////////////////////////////////////
+// Name:        tests/controls/infobar.cpp
+// Purpose:     wxInfoBar  tests
+// Author:      Blake Madden
+// Created:     2025-6-02
+// Copyright:   (c) 2025 Blake Madden
+///////////////////////////////////////////////////////////////////////////////
+
+// ----------------------------------------------------------------------------
+// headers
+// ----------------------------------------------------------------------------
+
+#include "testprec.h"
+
+
+#ifndef WX_PRECOMP
+    #include "wx/app.h"
+#endif // WX_PRECOMP
+
+#include "wx/infobar.h"
+
+#include <memory>
+
+namespace
+{
+
+    TEST_CASE("wxInfoBar", "[wxInfoBar]")
+    {
+        SECTION("Buttons")
+        {
+            const std::unique_ptr<wxInfoBar>
+                info(new wxInfoBar(wxTheApp->GetTopWindow(), wxID_ANY, wxINFOBAR_CHECKBOX));
+
+            CHECK(0 == info->GetButtonCount());
```

Comment:
Just FYI, you don't need to use Yoda comparison with CATCH.

Reply:
Sorry, old habit :)